libs/blocks/share/share.css

```diff
@@ -119,3 +119,72 @@
 .share .copy-to-clipboard.copy-to-clipboard-copied::before {
   content: attr(data-copied);
 }
+
+.share.inline {
+  margin: 0 auto;
+  text-align: center;
+  padding: 8px 0 16px 0;
+}
+
+.share.inline p.icon-container {
+  justify-content: flex-start;
+  width: fit-content;
+  margin: 0;
+}
+
+main > .section > .share.inline {
+  margin: 0 auto;
+}
+
+.center .share.inline p.icon-container {
+  margin: 0 auto;
+  justify-content: center;
+}
+
+main > .section > .inline-wrapper {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 8px 0 12px;
+}
```

Comment:
Still have a duplicate of this in `reading-time.css`.
If that container is only applied when there are inline siblings I think it makes sense to only include this snippet of css in one place. This will help avoid confusion in the event something needs to change or be adapted with this.

![image](https://user-images.githubusercontent.com/2539954/224180338-8ce38387-7cfe-43be-8e88-67abea1b788c.png)